CMSgov/onemac
# OY2 19727: Change Blue Text on Forms - Users Can Submit

Story: https://qmacbis.atlassian.net/browse/OY2-19727
Endpoint: https://dy9nb12dtm2s0.cloudfront.net/

### Details
Blue warning message text was updated by the help desk and approved by CMS to hopefully give better guidance.

### Changes
- updated the blue warning text
- updated the unit tests that were dependent
- discovered that the warning messages were not being included in all of the CMS emails, so added them to all of the emails.
- NOTE: I added the warning message to the Appendix K Amendment email, but I don't think the App K updates have been made yet?  

### Test Plan
1. Login as a submitting user
2. Navigate to Dashboard
3. Click New Submission->State Plan Amendment (SPA)->Respond to Formal CHIP SPA RAI
a. Verify that a SPA ID that exists on the Dashboard has no error/warning message
b. Verify that a SPA ID that does not exist on the Dashboard displays the warning message, note that there is no longer any format checking for CHIP SPA RAIs, so the existence check starts at the third character.
c. Verify the warning message is the one described in the AC
d. Verify you can successfully submit the form with the warning message showing.
e. Verify the email notice send to CMS contains the warning message. (Kristin can show you the email)
4. Click New Submission->State Plan Amendment (SPA)->Respond to Formal Medicaid SPA RAI
a. Verify that a SPA ID that exists on the Dashboard has no error/warning message
b. Verify that a SPA ID that does not exist on the Dashboard displays the warning message, note that there is no longer any format checking for Medicaid SPA RAIs, so the existence check starts at the third character.
c. Verify the warning message is the one described in the AC
d. Verify you can successfully submit the form with the warning message showing.
e. Verify the email notice send to CMS contains the warning message. (Kristin can show you the email)
5. Click New Submission->Waiver Action->Waiver Action, and Action Type choice "Request for waiver renewal"
a. Verify that you get an error message if the format does not match SS-####.R##.00 or SS-#####.R##.00
b. Verify that a duplicated Waiver Renewal Number also gets an error message
c. Verify that a non-duplicated Waiver Renewal Number with a matching Waiver Base has no message
d. Verify that a non-duplicated Waiver Renewal Number with no matching Waiver Base shows the warning 
e. Verify the warning message is the one described in the AC
f. Verify you can successfully submit the form with the warning message showing.
g. Verify the email notice send to CMS contains the warning message. (Kristin can show you the email)
11. Click New Submission->Waiver Action->Waiver Action, and Action Type choice "Waiver amendment"
a. Verify that you get an error message if the format does not match SS-####.R##.## or SS-#####.R##.##
b. Verify that a duplicated Waiver Amendment Number also gets an error message
c. Verify that a non-duplicated Waiver Amendment Number with a matching Waiver Base or Waiver Renewal has no message
d. Verify that a non-duplicated Waiver Renewal Number with no matching Waiver Base  or Waiver Renewal shows the warning 
e. Verify the warning message is the one described in the AC
f. Verify you can successfully submit the form with the warning message showing.
g. Verify the email notice send to CMS contains the warning message. (Kristin can show you the email)
13. Click New Submission->Waiver Action->Respond to Waiver RAI
a. Verify that a Waiver Number that exists on the Dashboard has no error/warning message
b. Verify that a Waiver Number that does not exist on the Dashboard displays the warning message, note that there is no longer any format checking for Waiver RAIs, so the existence check starts at the third character.
c. Verify the warning message is the one described in the AC
d. Verify you can successfully submit the form with the warning message showing.
e. Verify the email notice send to CMS contains the warning message. (Kristin can show you the email)
